src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
-
+import java.util.Random;
+import java.lang.String;
 public class ColorSupplier {
+    public final Random randomColor = new Random();
```

Comment:
Consider making `randomColor` a private variable since it is only used within the class.